describegroups.go

```diff
@@ -69,6 +69,18 @@ type DescribeGroupsResponseMemberMetadata struct {
 
 	// UserData is the user data for the member.
 	UserData []byte
+
+	// OwnedPartitions contains the partitions owned by this group member; only set if
+	// consumers are using a cooperative rebalancing assignor protocol.
+	OwnedPartitions []DescribeGroupsResponseMemberMetadataOwnedPartition
+}
+
+type DescribeGroupsResponseMemberMetadataOwnedPartition struct {
```

Comment:
Maximum Javaname intensity achieved :trollface:

Reply:
Haha, yes! Let's hope that the protocol isn't modified to add another layer of nesting, otherwise we're really going to be in trouble.